src/modules/geometry.cxx

```diff
@@ -0,0 +1,28 @@
+module;
+
+export module Geometry;
+
+export class Vector_2d
+{
+public:
+    double x_component, y_component;
```

Comment:
[tiny] I'm not sure `_component` adds too much to the understanding, I would go with simply `x` and `y` as members. What do you think?

Reply:
This would also keep it consistent with the point